ironfish/src/network/peers/peerConnectionManager.ts

```diff
@@ -110,21 +110,11 @@ export class PeerConnectionManager {
   }
 
   private connectToEligiblePeers(peer: Peer): boolean {
-    if (peer.state.type !== 'CONNECTED') {
-      if (this.peerManager.canConnectToWebRTC(peer)) {
```

Comment:
canConnectToWebRTC is called in connectToWebRTC

Reply:
<img width="885" alt="image" src="https://github.com/iron-fish/ironfish/assets/13268167/9d3ded97-49f3-4e22-8706-fca88e69ac0a">